transport/src/main/java/io/netty/channel/DefaultMaxMessagesRecvByteBufAllocator.java

```diff
@@ -50,13 +51,19 @@ public MaxMessagesRecvByteBufAllocator maxMessagesPerRead(int maxMessagesPerRead
     /**
      * Focuses on enforcing the maximum messages per read condition for {@link #continueReading()}.
      */
-    public abstract class MaxMessageHandle implements Handle {
+    public abstract class MaxMessageHandle implements ExtendedHandle {
         private ChannelConfig config;
         private int maxMessagePerRead;
         private int totalMessages;
         private int totalBytesRead;
         private int attemptedBytesRead;
         private int lastBytesRead;
+        private final UncheckedBooleanSupplier defaultMaybeMoreSupplier = new UncheckedBooleanSupplier() {
+            @Override
+            public boolean get() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)